packages/agoric-cli/tools/getting-started.js

```diff
@@ -105,121 +102,31 @@ export const gettingStartedWorkflowTest = async (t, options = {}) => {
-          startResult.resolve(true);
-        }
-      });
-    }
-    startP.childProcess.on('close', code =>
-      startResult.reject(Error(`early termination: ${code}`)),
-    );
+    // yarn start:docker
+    t.is(await yarn(['start:docker']), 0, 'yarn start:docker works');
 
-    const timeout = setTimeout(
-      startResult.reject,
-      TIMEOUT_SECONDS * 1000,
-      'timeout',
-    );
-    t.is(await startResult.promise, true, `swingset running before timeout`);
-    clearTimeout(timeout);
-
-    const openP = myMain(['open', '--no-browser'], {
-      stdio: ['ignore', 'pipe', 'inherit'],
-    });
-    t.is(await openP, 0, `open --no-browser exits successfully`);
-
-    const testDeploy = async (deployCmd, opts = {}) => {
-      const deployResult = makePromiseKit();
-      const deployP = myMain(
-        ['deploy', `--hostport=127.0.0.1:${PORT}`, ...deployCmd],
-        {
-          stdio: [opts.stdin ? 'pipe' : 'ignore', 'pipe', 'inherit'],
-        },
-      );
-
-      if (opts.stdin) {
-        // Write the input to stdin.
-        deployP.childProcess.stdin.write(opts.stdin);
-        deployP.childProcess.stdin.end();
-      }
-
-      finalizers.push(() => pkill(deployP.childProcess, 'SIGINT'));
-      const to = setTimeout(
-        deployResult.resolve,
-        TIMEOUT_SECONDS * 1000,
-        'timeout',
-      );
-      const done = await Promise.race([deployResult.promise, deployP]);
-      t.is(done, 0, `deploy ${deployCmd.join(' ')} successful before timeout`);
-      clearTimeout(to);
-    };
+    // TODO: use abci_info endpoint to get block height
+    // sleep for 180 seconds for now
+    await new Promise(resolve => setTimeout(resolve, TIMEOUT_SECONDS));
 
     // ==============
-    // agoric deploy ./contract/deploy.js ./api/deploy.js
-    await testDeploy(['./contract/deploy.js', './api/deploy.js']);
-
-    for (const [suffix, code] of [
-      ['/notthere', 404],
-      ['', 200],
-    ]) {
-      let urlResolve;
-      const url = `http://127.0.0.1:${PORT}${suffix}`;
-      const urlP = new Promise(resolve => (urlResolve = resolve));
-      const urlReq = request(url, res => urlResolve(res.statusCode));
-      urlReq.setTimeout(2000);
-      urlReq.on('error', err => urlResolve(`Cannot connect to ${url}: ${err}`));
-      urlReq.end();
-      const urlTimeout = setTimeout(urlResolve, 3000, 'timeout');
-      const urlDone = await urlP;
-      clearTimeout(urlTimeout);
-      t.is(urlDone, code, `${url} gave status ${code}`);
-    }
+    // yarn start:contract
+    t.is(await yarn(['start:contract']), 0, 'yarn start:contract works');
 
     // ==============
-    // cd ui && yarn start
-    const uiStartP = pspawn(`yarn`, ['start'], {
-      stdio: ['ignore', 'inherit', 'inherit'],
-      cwd: 'ui',
-      env: { ...process.env, PORT: '3000' },
-      detached: true,
-    });
+    // yarn start:ui
+    const uiStartP = yarn(['start:ui']);
     finalizers.push(() => pkill(uiStartP.childProcess, 'SIGINT'));
```

Comment:
_question_

I don't know if I need to add finalizers for other `yarn` commands above?